### DISCUSSION TITLE
Ponzi scheme debates

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#154] [General]
I note that ponzi scheme point comes up again and again and also gets dismissed again and again. E.g. here is Bloomberg piece on Terra https://www.bloomberg.com/news/features/2022-04-19/terraform-s-do-kwon-s-huge-bitcoin-buys-catch-crypto-billionaires-attention

> Others, including an anonymous Twitter user known as @AlgodTrading, who made one of the high-stakes bets with Kwon about the price of Luna, have compared the project to a Ponzi scheme because it requires demand for the tokens to keep growing.
>
> Of course, accusations of Ponzi schemes are pretty common in crypto, where there is typically no real-world cash flow or assets to back up the numbers on the screen. Bitcoin has been accused of being a Ponzi pretty much since its birth. Thirteen years and some $800 billion in market value later, Bitcoin is still going strong.

It would be worth really working through this point. What do we mean exactly by Ponzi scheme. How do we evaluate if X is a ponzi scheme,  and if X is a ponzi scheme when will it unwind?

**Comment by SalBayat** [comment]
Sorry for the late response, I have been working through your question about Ponzi schemes and will post my thoughts in a subsequent reply. 

### General Thoughts

Crypto narratives and propaganda have planted many false assumptions in the general public's mind.  The Bloomberg piece uncritically uses many of these presuppositions as facts, however, there is a more serious issue here.  The person who wrote this article is a true believer.  

The article is a puff piece written by someone who is heavily indoctrinated, and likely has a vested interest in the crypto industry.  Adherence to the ideology of crypto prevents both author and reader from evaluating claims from any other viewpoint aside from those that are officially sanctioned.  

Working through a cogent definition of a Ponzi and its characteristics is worthwhile.  It will help us form better arguments, and be clearer when we communicate with the public about the fundamental problems with crypto.  However, even the most perfect definition of a Ponzi scheme, combined with brilliant arguments will not help in this specific case.  Combating the Bloomberg article, and the many others like it, requires the use of a different strategy.

For the sake of brevity, I won't go through the entire article here but there are over a dozen sections that exemplify the lack of ethics in crypto journalism.  The most egregious is perhaps this one:

- _"Of course, accusations of Ponzi schemes are pretty common in crypto, where there is typically no real-world cash flow or assets to back up the numbers on the screen. Bitcoin has been accused of being a Ponzi pretty much since its birth. Thirteen years and some $800 billion in market value later, Bitcoin is still going strong."_

This is truly stunning.  Yes the emperor has no clothes, but have you seen how well he's dressed!  I am not sure if this is done intentionally, or if it's just been internalized by true believers, but they literally describe a Ponzi, and then imply it's not a Ponzi because the Ponzi is 'going strong'.  Crypto believers are inoculated against reason, the definition of a Ponzi scheme is redefined as normal, it's just how things are, this logical inconsistency does not need to be explained or scrutinized.  

Crypto Market Cap is a lie, its not 800B of value, its 800B of numbers on a screen, nowhere near that amount of liquidity could ever be realized because tokens were purchased at many different prices, but they calculate market cap by taking the latest price and multiplying it by the number of tokens.  A system that doesn’t generate revenue will be unable to fill the gap (close the delta) between the supposed liquidity available inside a cryptocurrency (or even the real total amount that has been invested), and the actual liquidity that can be withdrawn. 

### Fighting Propaganda and Disinformation

The author of the article is https://twitter.com/MuyaoShen, who appears to be very pro-crypto, and who was formerly @Coindesk ([Michael P. Regan](https://www.bloomberg.com/authors/AOiGCdwfgoM/michael-p-regan) likely served in an editorial capacity and reviewed the article).  The author was always going to portray the Crypto industry in a favourable light and dismiss concerns relating to its Ponzi like nature. 

Although the tide may be turning, as the stablecoin UST has crashed, more needs to be done to challenge crypto narratives in the mass media.  

1. Outreach to any journalists and publications which cover finance, with the goal of having articles critical of crypto published.  

2. Signed petitions by technology professionals/economists/journalists/experts opposing cryptocurrency.

3. Challenging news organizations who publish crypto puff pieces on the basis of journalistic integrity.  Social media should be used to pressure Bloomberg, or other media outlets to retract articles which are advertisements for the crypto industry. 

4. I think something like Molly White’s https://www.mollywhite.net/annotations/latecomers-guide-to-crypto which analyses and critiques these puff pieces.  This should be done more often, perhaps once a week, or once a month.  I would love to be involved in a regular segment that hacks away at the lies and misrepresentations in these articles.  There certainly are enough propaganda pieces published by major news outlets.  These critiques can then be used to challenge news outlets to produce more critical perspectives. 

5. Form some type of coalition, or organization, perhaps Life Itself is the right entity, or maybe its the EFF.  Call the initiative Truth in Technology.  Clearly outline its goal or mandate ex. "To promote democracy in technology".  Promote it so it has reach and an online presence, have all the crypto skeptics, critical journalists, technologists etc. join it.  Use this combined strength to push back on Crypto narratives.  Have a twitter account that warns the public of misleading articles, or crypto propaganda.  

     Crypto can align interests very well though the pursuit of Value, as such, although the industry is composed of disparate participants, they can all push in the same direction and because they all want the same thing, this allows them to form what is essentially a very powerful special interest group.  

     Opponents and Critics need to join together around the common interest of combating the spread of this harmful technology.  This will allow critical voices to be heard, and represented.  We need to influence those who are in positions of power and authority, and not just those who are already skeptical of crypto.  The solution here is the same as for other social ills, we need organization and collective action.  Even if Crypto has been weakened, there are always those who will look to technology to dominate and exploit others, they must be opposed. 

Unfortunately the game that is being played here is not one of presenting ideas, and have honest, rational debates.  Crypto is in the propaganda game, so the best strategies that we can use are the ones that combat disinformation in other arenas.

**Reply by rufuspollock** [reply]
Thanks for the detailed reflections.

What do you think of finding a definition of ponzi scheme and why or why not a given crypto project would resemble a ponzi scheme

**Reply by SalBayat** [reply]
I think I have a good answer for this question, but I am writing an article on it to answer it properly.  I will post it here once complete for review and comments, it may be of some small interest to you and others.

**Reply by rufuspollock** [reply]
Great.
